website/blog/2024-02-08-kubernetes-frontends/index.mdx

```diff
@@ -0,0 +1,199 @@
+---
+slug: kubernetes-frontends
+title: The Inner Workings of Kubernetes Management Frontends — A Software Engineers’ Perspective
+authors: [ christophenne ]
+tags: [ glasskube, kubernetes, frontends ]
+---
+
+*In this blogpost we are reviewing Kubernetes Management Frontends and discuss how these tools are being built.*
+
+# The Inner Workings of Kubernetes Management Frontends — A Software Engineers’ Perspective
```

Comment:
I think it should be "A Software Engineer's Perspective"?

Not sure because both would be grammatically correct but I think engineer (signular) would be more common.